src/index.html

```diff
@@ -15,8 +15,65 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header id="top">
+      <div class="imgContainer">
+        <a href="#top">
+          <img
+            class="img"
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#1"
+              class="is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li><a href="#2">samsung</a></li>
+          <li><a href="#3">smartphones</a></li>
+          <li>
+            <a
+              href="#4"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li><a href="#5">gadgets</a></li>
+          <li><a href="#6">tablets</a></li>
+          <li><a href="#7">photo</a></li>
+          <li><a href="#8">video</a></li>
```

Comment:
Fix formatting 
![image](https://github.com/user-attachments/assets/e8c82487-05dc-484a-aca9-279a03b5b6d3)

Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/user-attachments/assets/c770a410-eba0-4358-8ccb-b03f3bb550e3)